package.json

```diff
@@ -74,11 +74,6 @@
                 "title": "%azureResourceGroups.logIn%",
                 "category": "Azure"
             },
-            {
-                "command": "azureResourceGroups.logOut",
```

Comment:
I think we should still expose these commands. Anyone familiar to using them via Azure Accounts may be wondering where they are. Otherwise the only entry points for logging out is through the accounts view.

Reply:
Never mind, forgot about the resolved command.